src/components/Story.jsx

```diff
@@ -1,32 +1,45 @@
 import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
-import { fetchArticleById } from "../utils/api";
+import { fetchArticleById, fetchArticleComments } from "../utils/api";
 import { formatDate } from "../utils/helpers";
-import { Heading, Flex, Center, Text, Box } from "@chakra-ui/react";
+import { Heading, Flex, Text, Box } from "@chakra-ui/react";
+import CommentCard from "./CommentCard";
 
 const Story = () => {
     const [story, setStory] = useState({})
+    const [comments, setComments] = useState([])
     const { article_id } = useParams();
 
     useEffect(() => {
         fetchArticleById(article_id).then(({ article }) => {
-            setStory(article)
+            setStory(article);
+        })
+        fetchArticleComments(article_id).then(({comments}) => {
+            setComments(comments);
         })
     },[])
 
     return (
-        <Center>
+        <Flex as="main" tabIndex={0} direction="column" align="center" >
             <Flex as="article" direction="column" m={4} maxW="70%" alignItems="center" >
-                <Heading fontSize="2xl" textAlign="center" >{story && story.title}</Heading>
+                <Heading tabIndex={0} fontSize="2xl" textAlign="center" >{story && story.title}</Heading>
                 <Text>By <Box as="strong">{story.author}</Box> on <Box as="strong">{formatDate(story.created_at)}</Box></Text>
                 <Text>Category: {`${story.topic}`} </Text>
-                <Text my={2} maxW="80%" >{story.body}</Text>
+                <Text my={2} maxW="80%" minW="320px" >{story.body}</Text>
                 <Flex w="100%" justify="space-around" >
                     <Text>Votes: {`${story.votes}`}</Text>
                     <Text>Comments: {`${story.comment_count}`}</Text>
                 </Flex>
             </Flex>
-        </Center>
+            <Flex as="section" direction="column" tabIndex={0} align="center" w="50%" >
+                <Heading as="h3" fontSize="lg" >Comments</Heading>
+                {
+                    comments.map((comment) => (
+                        <CommentCard key={comment.comment_id} comment={comment} />
```

Comment:
It would be nice to have some loading state here whilst we're waiting for the comments to come back. You may also want to consider what should be displayed if there are no comments? (like for article number 37 🙂 )